api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -458,4 +459,13 @@ public void updateWorkspaceBillingAccount(DbWorkspace workspace, String newBilli
                     .build())
         .collect(ImmutableList.toImmutableList());
   }
+
+  @Override
+  public Boolean notebookTransferCompleted(String workspaceNamespace, String workspaceId) {
+    String fileTransferTime =
```

Comment:
curious why this returns String - all we want to know is whether there's a valid time here

Reply:
Yeah True or false would have been ideal, but somehow Firecloud is sending dateTime

Reply:
OK.  What do you think about `getDuplicateWorkspaceFileTransferTime()` returning a boolean instead?  (we should also use primitive-boolean wherever possible)